### PR TITLE
Support creating multiple API instances

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: code/main/task/taskNode.brs [hunk 1]
```diff
@@ -20,6 +20,18 @@ function _adb_createTaskNode() as void
     end if
 end function
 
+function _adb_storeTaskNode(taskNode as object) as void
+    if GetGlobalAA()._adb_main_task_node = invalid
+        GetGlobalAA()._adb_main_task_node = taskNode
+    end if
+end function
+
+function _adb_setTaskNode(taskNode) as void
```
Comment (praveek) [inline code/main/task/taskNode.brs:29]: Where is this function used?

Comment (yangyansong-adbe) [reply code/main/task/taskNode.brs:29]: Oh, it's merged from the POC branch, I should remove it.

File: code/AEPSDK.brs [hunk 2]
```diff
@@ -36,17 +36,40 @@ end function
 
 ' *****************************************************************************
 '
-' Initialize the Adobe SDK and return the public API instance.
-' The following variables are reserved to hold SDK instances in GetGlobalAA():
+' The AEP task node performs the core logic of the SDK.
+' If the task node instance is not provided, calling this function will create a
```
Comment (addb) [inline code/AEPSDK.brs:40]: Let's reword to something like this:

TaskNode should not provided while initializing SDK if you plan to use this in the MainScene and don't need to create multiple instances of the SDK. If you need to create instances for different screen then you can use getTaskNode API to get the taskNode in the MainScene and use that taskNode to create SDK instance in the other scenes.

**Review by addb — APPROVED** [review]
Looks good after all the suggested and discussed changes!

File: sample/development/components/MainScene.brs [hunk 3]
```diff
@@ -35,7 +35,9 @@ sub _initSDK()
   m.aepSdk = AdobeAEPSDKInit()
   print "Adobe SDK version : " + m.aepSdk.getVersion()
   m.adobeTaskNode = m.aepSdk.getTaskNode()
-  m.adobeTaskNode.id = "adobeTaskNode"
+  ' The task node has a default id => "adobeTaskNode"
+  ' If you want to set it to anther value, you can enalbe the below code
```
Comment (praveek) [inline sample/development/components/MainScene.brs:39]: enable

**Review by praveek — APPROVED** [review]

**Pull request merged**